homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>PokemonProject</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="./style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav justify__content--center">
+                <div class="container justify__content--between">
+                    <div class="nav__logo">
+                        <img class="logo__icon" src="./assets/logo-icon.svg" alt="logo">
+                        <img class="logo__title" src="./assets/name-icon.svg" alt="name">
+                    </div>
+                    <ul class="nav__menu">
```

Comment:
I see big distance between About and Pokemon links.  And I think header can have full width.
<img width="1434" alt="Screenshot 2023-10-23 at 12 36 32" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/115080771/11080ae5-fe16-48ff-afff-dbd70bc8051f">

Reply:
changed the distance and made header 100%